docs/mine/lotus/split-markets-miners.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Lotus Miner: Splitting main miner and markets service processes'
+description: ''
+breadcrumb: 'Splitting main miner and markets service processes'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+## Background
+
+Lotus v1.11.0 introduced the notion of `subsystems` in the `lotus-miner` process. Currently there are 4 subsystems, that can be enabled or disabled in the `config.toml` file:
+
+```
+[Subsystems]
+#  EnableMining = true
+#  EnableSealing = true
+#  EnableSectorStorage = true
+#  EnableStorageMarket = true
+```
+
+All these options are set to `true` by default. Until now, the monolith `lotus-miner` process has been responsible for all functionality.
+
+At the moment these `subsystems` are designed to be grouped into two distinct types of `lotus-miner` nodes:
+
+1. The `markets` node - a `lotus-miner` process responsible to handling the storage market subsystem, and all functionality related to storage/retrieval deals;
+
+```
+[Subsystems]
+  EnableMining = false
+  EnableSealing = false
+  EnableSectorStorage = false
+  EnableStorageMarket = true
+```
+
+2. The `mining/sealing/proving` node - a `lotus-miner` process responsible to Filecoin mining, and sector storage, sealing and proving;
+
+```
+[Subsystems]
+  EnableMining = true
+  EnableSealing = true
+  EnableSectorStorage = true
+  EnableStorageMarket = false
+```
+
+When a `lotus-miner` node is started with configuration for a `markets` node, it includes a libp2p node and should be publicly exposed on the Internet and reachable/dialable for other clients to communicate with it.
+
+When a `lotus-miner` node is started with configuration for a `mining/sealing/proving` node, it does not include a libp2p node, and it doesn't have to be publicly exposed on the Internet. The `markets` node communicates with it via its JSON RPC interface.
+
+This document explains how to split your existing monolith `lotus-miner` node into multiple processes and take advantage of this new architecture, which brings more security to your mining operation as well as less potential attack surface.
+
+## Pre-requisites
+
+Before splitting the markets service process from the monolith miner process, you should backup your miner's metadata repository. You need to start the `lotus-miner` with the `LOTUS_BACKUP_BASE_PATH` env variable in order to do that.
+
+```sh
+export LOTUS_BACKUP_BASE_PATH=~/lotus-backup-location
+lotus daemon
+```
+
+```sh
+export LOTUS_BACKUP_BASE_PATH=~/lotus-backup-location
+lotus-miner run
+```
+
+## Creating a backup
+
+```sh
+export LOTUS_BACKUP_BASE_PATH=~/lotus-backup-location
+lotus-miner backup ~/lotus-backup-location/backupfile
+```
+
+## Copy config.toml and storage.json for the markets service
+
+You need to generate a `config.toml` and `storage.json` for the markets node, and have them ready for the next step.
```

Comment:
Link to instructions?

Reply:
Can we also explain what each file does, as a reminder?

Reply:
I've linked to the two relevant pages, although I think it is fair to say that miners who attempt to split their monolith, should already be quite familiar with the `config.toml` file.